### PR TITLE
Bump version to v0.16.0-dev.0-5

### PR DESCRIPTION
**Pull request by sebastian-quintero**

### CODE REVIEW

**Review by larsbeck — APPROVED** [review]

**Review by davidrijsman — APPROVED** [review]

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-v0.16.0-dev.0-4
\ No newline at end of file
+v0.16.0-dev.0-5
```
Comment (merschformann) [inline VERSION:1]: No new line at end of file? :thinking:

**Review by merschformann — APPROVED** [review]
:rocket:

File: VERSION (hunk 1 above)
Comment (sebastian-quintero) [reply VERSION:1]: No, because that `VERSION` file is used to name the `.so` files, so we need to skip the newline.

Comment (merschformann) [reply VERSION:1]: Ah, makes sense. :+1:

**Pull request merged**